### ISSUE TITLE
Update pls :D

### ISSUE THREAD

**Issue by Gitterzwerg** [jaycewhite/MiniMappingway#29]
Needs update for current patch pls :D

**Comment by deansheather** [comment]
For what it's worth running a locally built dev version seems to work fine since 0ee0245d10fee5c45e6f5593ec0dda04312ad2c7, so I imagine this just needs to be submitted to the Dalamud repo

**Comment by jaycewhite** [comment]
Yeah, I am planning to submit sometime tomorrow, I just wanted to play the game with it a bit first to make sure it didn't cause issues 😄
